src/System.Diagnostics.EventLog/src/System/Diagnostics/Reader/EventLogRecord.cs

```diff
@@ -292,7 +292,14 @@ public override string FormatDescription(IEnumerable<object> values)
             {
                 if (theValues.Length == i)
                     Array.Resize(ref theValues, i + 1);
-                theValues[i] = o.ToString();
+                if (o is EventProperty elp)
```

Comment:
I just noticed the lines above -- it's reallocating the array on every single iteration!

No need to change today, but it ought to be a `List<string>`. Maybe it's super rare there are more than 1 or 2 objects.

Reply:
Yes, I noticed that too.